src/Component/DataInput/DataLoader/DataLoader.tsx

```diff
@@ -102,6 +102,28 @@ export class DataLoader extends React.Component<DataLoaderProps, DataLoaderState
     };
   }
 
+  parseShapefileUploadData = (uploadObject: any) => {
```

Comment:
Do we have any type info for the `uploadObject`?

Reply:
I added the missingt typings.